database/seeders/DoctorSeeder.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use Database\Factories\DoctorFactory;
+use Illuminate\Database\Seeder;
+
+class DoctorSeeder extends Seeder
+{
+    public function run(): void
+    {
+        $factory = new DoctorFactory();
+        $factory->createMany(10);
```

Comment:
```suggestion
        DoctorFactory::new()->createMany(10);
```
Any particular reason why you chose to perform the create this way?

Reply:
Really I do not remember :(